package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-plugin-lambda-account-access",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
Should we bump this to `3.0.0`? A project using an older version of serverless would break if it picked up this change, right?

Reply:
Good point. Updated min `serverless` version as well.